### PR TITLE
fix and activate E225 and E228

### PR DESCRIPTION
**Pull request by fchapoton**
This is fixing (around 200 cases) and activating pycodestyle E225 and E228 in the linter

```
E228 missing whitespace around modulo operator
E225 missing whitespace around operator
```

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

**Comment by fchapoton** [comment]
the CI seems to be broken, but the changes here should be harmless

File: src/sage/interacts/library.py [hunk 1]
```diff
@@ -1327,36 +1327,36 @@ def parabola(a, b, c):
...
-    html(r'Integral value to seven decimal places is: $\displaystyle\int_{%.2f}^{%.2f} {f(x) \, \mathrm{d}x} = %.6f$'%
+    html(r'Integral value to seven decimal places is: $\displaystyle\int_{%.2f}^{%.2f} {f(x) \, \mathrm{d}x} = %.6f$' %
          (interval[0],interval[1],
          N(numeric_value,digits=7)))
 
     if output_form == 'traditional':
         sum_formula_html = r"\frac{d}{3} \cdot \left[ f(x_0) + %s + f(x_{%s})\right]" % (
-            ' + '.join([ r"%s \cdot f(x_{%s})" %(i%2*(-2)+4, i+1) for i in range(0,n-1)]),
+            ' + '.join([ r"%s \cdot f(x_{%s})" % (i % 2*(-2)+4, i+1) for i in range(0,n-1)]),
             n
```
Comment (kwankyu) [inline src/sage/interacts/library.py:1346]: Spaces around other operators too.

File: src/sage/interacts/library.py [hunk 2]
```diff
@@ -1327,36 +1327,36 @@ def parabola(a, b, c):
...
             )
 
         sum_placement_html = r"\frac{%.2f}{3} \cdot \left[ f(%.2f) +  %s + f(%.2f)\right]" % (
             dx,
             N(xs[0],digits=5),
-            ' + '.join([ r"%s \cdot f(%.2f)" %(i%2*(-2)+4, N(xk, digits=5)) for i, xk in enumerate(xs[1:-1])]),
+            ' + '.join([ r"%s \cdot f(%.2f)" % (i % 2*(-2)+4, N(xk, digits=5)) for i, xk in enumerate(xs[1:-1])]),
             N(xs[n],digits=5)
```
Comment (kwankyu) [inline src/sage/interacts/library.py:1354]: The same here

File: src/sage/interacts/library.py [hunk 3]
```diff
@@ -1327,36 +1327,36 @@ def parabola(a, b, c):
...
 
-        sum_values_html = r"\frac{%.2f}{3} \cdot \left[ %s %s %s\right]" %(
+        sum_values_html = r"\frac{%.2f}{3} \cdot \left[ %s %s %s\right]" % (
             dx,
-            "%.2f + "%N(ys[0],digits=5),
-            ' + '.join([ r"%s \cdot %.2f" %(i%2*(-2)+4, N(yk, digits=5)) for i, yk in enumerate(ys[1:-1])]),
-            " + %.2f"%N(ys[n],digits=5)
+            "%.2f + " % N(ys[0],digits=5),
+            ' + '.join([ r"%s \cdot %.2f" % (i % 2*(-2)+4, N(yk, digits=5)) for i, yk in enumerate(ys[1:-1])]),
+            " + %.2f" % N(ys[n],digits=5)
```
Comment (kwankyu) [inline src/sage/interacts/library.py:1362]: The same here too.

File: src/sage/interacts/library.py [hunk 4]
```diff
@@ -1377,14 +1377,14 @@ def parabola(a, b, c):
     elif output_form == 'table':
         s = [['$i$', '$x_i$', '$f(x_i)$', '$m$', r'$m\cdot f(x_i)$']]
         for i in range(0,n+1):
-            if i==0 or i==n:
+            if i == 0 or i == n:
                 j = 1
             else:
-                j = (i+1)%2*(-2)+4
+                j = (i+1) % 2*(-2)+4
             s.append([i, xs[i], ys[i],j,N(j*ys[i])])
```
Comment (kwankyu) [inline src/sage/interacts/library.py]: The same.

**Comment by kwankyu** [comment]
Otherwise, lgtm.

File: src/sage/interacts/library.py [hunk 5]
```diff
@@ -1376,12 +1379,12 @@ def parabola(a, b, c):
...
         s = [['$i$', '$x_i$', '$f(x_i)$', '$m$', r'$m\cdot f(x_i)$']]
-        for i in range(0,n+1):
+        for i in range(n + 1):
             if i == 0 or i == n:
                 j = 1
             else:
-                j = (i+1) % 2*(-2)+4
-            s.append([i, xs[i], ys[i],j,N(j*ys[i])])
+                j = (i + 1) % 2*(-2) + 4
+            s.append([i, xs[i], ys[i], j, N(j*ys[i])])
```
Comment (kwankyu) [inline src/sage/interacts/library.py:1387]: `2 * (-2)`

**Review by kwankyu — APPROVED** [review]
Thanks.

**Comment by vbraun** [comment]
merge conflict

**Comment by fchapoton** [comment]
setting back to positive after easy merge

**Pull request merged**